public/v1/apps/gitlab-runner.json

```diff
@@ -0,0 +1,30 @@
+{
+    "captainVersion": "1",
+    "documentation": "Taken from https://docs.gitlab.com/runner/install/docker.html and https://docs.gitlab.com/runner/register/",
+    "dockerCompose": {
+        "version": "3",
+        "services": {
+          "$$cap_appname": {
+            "image": "gitlab/gitlab-runner:$$cap_gitlab-runner_version",
+            "notExposeAsWebApp": "true",
+            "volumes": [
+              "$$cap_appname-data:/etc/gitlab-runner",
+              "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
Good catch. I personally often have to build docker images in my pipelines, and mounting the docker socket into the container is [one way](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#use-docker-socket-binding) that let's me do this. There are [other ways](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html) to achieve this though. If you mind, I'll remove the mount.

Reply:
That's fine. Just wanted to make sure that it's not added by accident. 

Let's just add a bit of description to `instructions.start` to point out that this container will have full access to all containers via a Docker socket binding. So that the users understand the potential risk.

Reply:
Done 👍🏻